docs/walkthrough.md

```diff
@@ -0,0 +1,551 @@
+
+We don't actually _have_ to specify an icon here; `ContentBase` from which `NewsFeed` inherits has a default icon.
+Our news app is super-special, though, so let's give it an icon all its own.
+At Onespacemedia we made a whole lot of icons, all in the same style, which are perfect for the 'Add a page' screen.
+For now, go grab
+[this one](https://github.com/onespacemedia/cms-icons/blob/master/png/news.png)
+for your news app and put it in your `static/icons` directory.
+
+This content model will be a news feed, to which articles can be assigned (with a normal `ForeignKey`).
+We want to be able to have multiple types of news feed.
+For example, we might want to have a page of articles called "News" (what your cat did today) vs "Blog" (insights on cat behaviour).
+We'll get to exactly how this will happen shortly.
+But first, we're going to need an Article model.
+
+## Let's use the CMS's helper models
+
+The CMS comes with a lot of handy helper models, and some nice helper fields too.
+You will want to use them, because you should always use the batteries! We're going to be introducing a couple of them in our `Article` model.
+
+First, add these imports to your `news/models.py`:
+
+```python
+from cms.apps.media.models import ImageRefField
+from cms.models import HtmlField, PageBase
+```
+
+And add the model itself:
+
+```python
+class Article(PageBase):
+    page = models.ForeignKey(
+        'news.NewsFeed',
+        on_delete=models.PROTECT,
+        null=True,
+        blank=False,
+        verbose_name='News feed'
+    )
+
+    image = ImageRefField(
+        null=True,
+        blank=True,
+        on_delete=models.PROTECT,
+    )
+
+    content = HtmlField()
+
+    date = models.DateTimeField(
+        default=now,
+    )
+
+    summary = models.TextField(
+        blank=True,
+    )
+
+    class Meta:
+        ordering = ['-date']
+
+    def __str__(self):
+        return self.title
+```
+
+OK, let's talk about PageBase here.
+It's a helper (abstract) model to make it easier for you to have article-like fields on your model.
+It has nearly all the fields that the Page model itself does, but does not consider itself part of any hierarchy.
+In fact, the CMS Page itself inherits from PageBase.
+Here's what you get (see the [helper models](helpers.md) section for more):
+
+* A title and slug
+* Online/offline controls (enforced by the manager)
+* OpenGraph and Twitter card fields.
+* SEO fields like meta descriptions and a title override.
+
+On to `ImageRefField`.
+You can read about the [media app](media-app.md) later on, but the short version is: it's a model wrapper around Django's `FileField`.
+`ImageRefField` is a ForeignKey to `media.File`, but it uses a raw ID widget by default, and is constrained to only select files that appear to be images (just a regex on the filename).
+
+`HtmlField` is an HTML field with a nice WYSIWYG editor as its default widget - that was what the `WYSIWYG_OPTIONS` setting was about.
```

Comment:
Consider "`HtmlField` is a `TextField` with a nice WYSIWYG editor as its default widget"